Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -763,6 +763,11 @@ protected void IncreaseDimensionIndex()
             }
         }
 
+        public void ForceUpdateThemes()
```

Comment:
>  public void ForceUpdateThemes() [](start = 6, length = 33)

Nit: Add a doccomment (really about when it should be called and guidance around it)

More generally, would it make sense to have it so that setting interactable.Profiles would trigger this automatically?

Reply:
Second question is less important.

---
In reply to: [297268770](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5033#discussion_r297268770) [](ancestors = 297268770)

Reply:
I think we should open a feature request issue for interactable runtime support in general. Changing themes and changing IsGlobal would both fit there, along with a static utility class for assembling themes.